contrib/copy-tex/Makefile

```diff
@@ -1,15 +0,0 @@
-.PHONY: build
-
-build: $(BUILDDIR)/contrib/copy-tex.min.js $(BUILDDIR)/contrib/copy-tex.css $(BUILDDIR)/contrib/copy-tex.min.css
-
-$(BUILDDIR)/contrib/copy-tex.min.js: $(BUILDDIR)/contrib/copy-tex.js
-	$(UGLIFYJS) < $< > $@
-
-$(BUILDDIR)/contrib/copy-tex.js: copy-tex.js
-	$(BROWSERIFY) -t [ babelify ] $< --standalone renderMathInElement > $@
```

Comment:
Is exporting the module to `renderMathInElement` an intended behavior?

Reply:
Thanks for catching this.  I guess nobody's been using this as a standalone script.